.github/workflows/cd.yaml

```diff
@@ -0,0 +1,24 @@
+name: CD
+
+on:
+  push:
+    tags: ["v*"]
```

Comment:
Our release tags so far have not started with `v`, see: https://github.com/zappa/Zappa/tags.  Perhaps we could match on semver regex instead?  `\d+\.\d+\.\d+`

Per the [docs](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet), it seems the GHA equivalent would be: `[0-9]+.[0-9]+.[0-9]+`

Reply:
I meant to bring this up on Slack, as it does seem the convention [changed at some point](https://github.com/Miserlou/Zappa/tags).

Reply:
I've at least changed it to the more explicit `v[0-9]+.[0-9]+.[0-9]+` for now.

Reply:
Should we support both to be safe?  I think just adding a `?` after the `v` should do it:  `v?[0-9]+.[0-9]+.[0-9]+`

Reply:
good idea, done 👍